sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client_models.hpp

```diff
@@ -292,6 +299,39 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
     AZ_SECURITY_KEYVAULT_KEYS_DLLEXPORT static const KeyCurveName P521;
   };
 
+  /**
+   * @brief The encryption algorithm to use to protected the exported key material.
```

Comment:
This comment is coming from the swagger, but we should fix it to be worded correctly:
https://github.com/Azure/azure-rest-api-specs/blob/37cd8dfac3c570a24bb645b31c012d12efb760df/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.3/keys.json#L1772

```suggestion
   * @brief The encryption algorithm to use to protect the exported key material.
```